irohad/ametsuchi/impl/redis_block_query.cpp

```diff
@@ -170,6 +170,21 @@ namespace iroha {
           });
     }
 
+    rxcpp::observable<model::Transaction> RedisBlockQuery::getTransactions(
+        const std::vector<iroha::hash256_t> &tx_hashes) {
+      return rxcpp::observable<>::create<model::Transaction>(
+          [this, tx_hashes](auto subscriber) {
+            std::for_each(
+                tx_hashes.begin(),
+                tx_hashes.end(),
+                [ that = this, &subscriber ](auto tx_hash) {
```

Comment:
extra space after/before square brackets :(

Reply:
`clang-format` does this ;(